src/spatio_temporal_voxel_layer.cpp

```diff
@@ -479,6 +523,42 @@ bool SpatioTemporalVoxelLayer::RemoveStaticObservations(void)
   }
 }
 
+/*****************************************************************************/
+void SpatioTemporalVoxelLayer::DynamicReconfigureCallback( \
```

Comment:
This might be my ignorance of dynamic reconfigure, but does it send params even if they don't change? ie to fill out the `config` message for variable `x` does it send `y` as the default value even though you didnt want to change it? Otherwise some of these variables set below would be consistently overwritten if you just wanted to enable/disable through this mechanism

Reply:
For what I know:
Dynamic reconfigure at the init first loads the dynamic reconfigure.cfg then its values are overwritten by ros_params (commom_params.yaml). 
When in run time some params are changed with the dynamic reconfigure, all of them are updated. However the value is updated with the dynamic reconfigured config which is the one obtained from ros_params so I wont be any problems. I'll test it anyway.
I hope I explained clearly...

Reply:
Ok sounds good - given that the callback runs after the ROS params are evaluated (in the first iteration) I would think it would overwrite, unless we did the dynamic reconfigure callback registering **before** we call ros params (we might want to just move it so there's no chance). 

Thanks for the explanation

Reply:
Tested worked as expected! params are not everrided.